Documentation/CRDs/Block-Storage/ceph-block-pool-crd.md

```diff
@@ -28,41 +34,30 @@ spec:
 
-Hybrid storage is a combination of two different storage tiers. For example, SSD and HDD.
-This helps to improve the read performance of cluster by placing, say, 1st copy of data on the higher performance tier (SSD or NVME) and remaining replicated copies on lower cost tier (HDDs).
+This example specifies a pool with an EC 4+2 data protection scheme.  For a
+given amount of raw storage, this pool yields double the usable space compared
+to the replicated ``size 3`` example above.  Redundancy is provided by
+specifying an [erasure coding](#erasure-coding) profile with ``codingChunks: 2``,
+akin to a conventional RAID6 volume but with much greater flexibility.  Data
+remains available if any one node is down and is preserved if any two are lost.
 
-**WARNING** Hybrid storage pools are likely to suffer from lower availability if a node goes down. The data across the two
-tiers may actually end up on the same node, instead of being spread across unique nodes (or failure domains) as expected.
-Instead of using hybrid pools, consider configuring [primary affinity](https://docs.ceph.com/en/latest/rados/operations/crush-map/#primary-affinity) from the toolbox.
-
-```yaml
-apiVersion: ceph.rook.io/v1
-kind: CephBlockPool
-metadata:
-  name: replicapool
-  namespace: rook-ceph
-spec:
-  failureDomain: host
-  replicated:
-    size: 3
-    hybridStorage:
-      primaryDeviceClass: ssd
-      secondaryDeviceClass: hdd
-```
-
-!!! important
-    The device classes `primaryDeviceClass` and `secondaryDeviceClass` must have at least one OSD associated with them or else the pool creation will fail.
-
-### Erasure Coded
-
-This sample will lower the overall storage capacity requirement, while also adding redundancy by using [erasure coding](#erasure-coding).
+The downside is that client write performance will be roughly half that of the
+above replicated pool, which notably means longer recovery (and thus increased
+risk) when a server or OSD drive fails.
 
 !!! note
-    This sample requires *at least 3 bluestore OSDs*.
+    This example requires *at least* six OSDs*.
 
-The OSDs can be located on a single Ceph node or spread across multiple nodes, because the [`failureDomain`](ceph-block-pool-crd.md#spec) is set to `osd` and the `erasureCoded` chunk settings require at least 3 different OSDs (2 `dataChunks` + 1 `codingChunks`).
+The OSDs can be located on a single Ceph node or spread across multiple nodes,
+because the [`failureDomain`](ceph-block-pool-crd.md#spec) is set to `osd`.
+The `erasureCoded` chunk settings require at least six OSDs of the specified
+deviceClass: four `dataChunks` + two `codingChunks`).  We provide this example
+of selecting the `osd` failure domain to demonstrate the idea, but for valuable
+production data this is very risky: a single host going unavailable will mean
+that at least some data is unavailable, and the complete loss of a single host's
+OSDs -- or *any* three OSDs -- will mean that some or all data is irretrievably lost.
```

Comment:
Ditto the italics/bold question for "any". To help ensure clarity of intent, could you use either `_italic_` or `**bold**`

```suggestion
OSDs -- or _any_ three OSDs -- will mean that some or all data is irretrievably lost.
```